addons/l10n_cz/views/report_template.xml

```diff
@@ -5,7 +5,8 @@
             Company ID: <span t-field="company.company_registry"/>
         </li>
         <li t-if="company.vat and company.account_fiscal_country_id.code == 'CZ'">
-            <t t-out="company.country_id.vat_label or 'Tax ID'"/>:
+            <t t-set="default_vat_label">Tax ID</t>
```

Comment:
Make sure to update the .pot file and translations for this (l10n module), and the one below.